Gems/ONNX/README.md

```diff
@@ -38,3 +38,139 @@ This is an experimental gem implementing [ONNX Runtime](https://onnxruntime.ai/)
 - [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit) v11.4 or greater.
 - [CUDNN library](https://developer.nvidia.com/cudnn) v8.2.4 or greater.
 - [zlib](https://zlib.net/) v1.2.3 or greater.
+
+## Additional developer docs
+
+### **What is ONNX?**
```

Comment:
update: We can probably remove the emphasis here in addition to the heading (using `###` should be sufficient)

Reply:
(repeated in a few spots)

Reply:
Removed